membersuite_api_client/tests/test_memberships.py

```diff
@@ -21,6 +24,14 @@ def test_get_membership_for_org(self):
                          '6faf90e4-0074-cbb5-c1d2-0b3c539859ef')
 
         # Test org without a membership
+
+        # Need an Organization ID for a non-member org. The one here
+        # is the same used above for testing with a member org.
+        #
+        #            HERE!
+        #              |
+        #              |
+        #              V
         test_org_id = "6faf90e4-0007-c9dc-98b7-0b3c53985743"
```

Comment:
These are actually different: '6faf90e4-0074-cbb5-c1d2-0b3c539859ef' != "6faf90e4-0007-c9dc-98b7-0b3c53985743"

The first is for some org that had a membership, the second is "AASHE Test Campus" that does not. We would of course change these if we ever get a sandbox set up.